### ISSUE TITLE
p-value of marginal boxplots of PCoA1 and PCoA2

### ISSUE THREAD

**Issue by NuruddinKhoiry** [alfbenpa/permubiome#1]
Hi,

I read your paper entitled "_Depletion of Blautia Species in the Microbiota of Obese Children Relates to Intestinal Inflammation and Metabolic Phenotype Worsening_" which led me to this GitHub website. 

I have a question related to Figure 1 of your paper. How did you calculate the p-value of the marginal boxplots of PCoA1 and PCoA2? I would like to perform a similar analysis. Could you please guide me on how to do so? Would you mind sharing the script used for it?

Best regards,
Khoiri

**Comment by alfbenpa** [comment]
Hi Khoiri,
You have to evaluate individually the components with a larger explained variation. From the multidimensional reduction method used (PCA, PCoA, NMDS, RDA, CCA, etc.), select the first two component values individually and evaluate their distribution according to your grouping variable. Depending on the distribution of component values (e.g. Shapiro-Wilk test), you should apply parametric or non-parametric tests on comparisons. T-test (two groups) or ANOVA (multiple groups) for normally distributed data OR Wilcoxon Rank Sum test (two groups) or Kruskal-Wallis (multiple groups) for non-normally distributed data.

Best regard,

**Comment by NuruddinKhoiry** [comment]
Thank you for the explanation. I understand better now. 
I have one more question. Since both axes (PC1 and PC2) have positive and negative values, how do you handle this? Do you need to transform the value before performing the statistical calculation?

Best regards,
Khoiri

**Comment by alfbenpa** [comment]
Not at all. You will explore distributions of values obtained, so you don't need to transform anything.

Best regards,
Alfonso

**Comment by NuruddinKhoiry** [comment]
Got it. Thank you very much for your explanation.

Best regards,
Khoiri
